docs/converters/tabular_converter.md

```diff
@@ -130,15 +128,12 @@ You can use the following `column` definitions:
     )
     ```
 ## Units
-Power Grid Model uses SI units (e.g. "W" for Watts),
-but source data may be supplied in different units (e.g. "MW" for Mega Watts).
-If units are supplied in the tabular data,
-the pandas DataFrame storing the data is expected to have `MultiIndexes` for the columns.
-For our application, a `MultiIndex` can be interpreted as a tuple; the first element is the column name, the second 
-element is the column unit. For example: `("C0", "µF")`.
+Power Grid Model uses SI units (e.g., "W" for Watts), but source data may be supplied in different units (e.g., "MW" for Mega Watts).
+If units are supplied in the tabular data, the data stored using pandas DataFrame is expected to have `MultiIndexes` for columns.
+For our application, a `MultiIndex` can be interpreted as a tuple; the first element is the column name, the second element is the column unit. For example: `("C0", "µF")`.
```

Comment:
Throughout the whole section I am still not sure whether units are required or not.

Reply:
I would say so, because we do present some kind of unit mapping interface (especially since not all tools use the same default unit conventions)